package.json

```diff
@@ -44,16 +44,22 @@
     "react-select": "1.3.0",
     "react-transition-group": "2.5.3",
     "reactstrap": "6.5.0",
+    "regenerator-runtime": "^0.13.7",
     "style-loader": "0.21.0",
     "url-loader": "1.0.1",
     "whatwg-fetch": "2.0.4"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.14.5",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.14.5",
     "ajv": "6.9.1",
     "asar": "0.14.6",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "8.2.6",
-    "babel-loader": "7.1.5",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Do we need both `babel-core` and `@babel/core`?

Reply:
Nope, I removed the former.